pkg/virt-controller/watch/migration_test.go

```diff
@@ -62,18 +61,13 @@ import (
 var _ = Describe("Migration watcher", func() {
 
 	var ctrl *gomock.Controller
-	var migrationPolicyInformer cache.SharedIndexInformer
-	var resourceQuotaInformer cache.SharedIndexInformer
-	var namespaceInformer cache.SharedIndexInformer
```

Comment:
These could be amended to previous commit. This commit could than handle only `migrationPolicyInformer`. Up to you.

Reply:
I squashed the commits all together. Does it work for you? Thank you